src/components/AdvancedFacetViewModal/index.js

```diff
@@ -54,6 +55,11 @@ class AdvancedFacetViewModalContent extends React.Component {
         <AfvContainer>
           <AdvancedFacetView
             {...props}
+            FilterInputComponent={props => (
+              <div>
```

Comment:
Is this <div> necessary? it looks like `FilterInput` is already wrapped in a `div`, and if it's not necessary then this could be `FilterInputComponent={FilterInput}`

Reply:
there was a bit of a styling issue that came from the theme's input style. Applied a better fix by overriding that style instead ;)